packages/fuel-indexer-tests/src/fixtures.rs

```diff
@@ -644,6 +644,28 @@ pub mod test_web {
         HttpResponse::Ok()
     }
 
+    async fn fuel_indexer_test_trigger_revert(
+        state: web::Data<Arc<AppState>>,
+    ) -> impl Responder {
+        let result = state
+            .contract
+            .methods()
+            .trigger_revert()
+            .tx_params(tx_params())
+            .call()
+            .await;
+
+        println!("Result: {:?}", result);
+
+        match result {
+            Ok(_) => HttpResponse::Ok(),
+            Err(_) => {
+                //Force 200 on err response here, because we have successfully triggered a panic
+                HttpResponse::Ok()
+            }
+        }
```

Comment:
Nice catch here! I think you can get away with removing the comment and result printing, and then matching on just the exhaustive case since we're doing the same action either way. For example:
```
    enum MysteryIncSounds {
        Zoinks,
        Jinkies,
        RuhRoh
    }
    
    let mystery_inc_sound = MysteryIncSounds::Zoinks;
    
    match mystery_inc_sound {
        _ => println!("Why are kids investigating crimes anyways?"),
    }
```

Reply:
This is a vivd example, love it.

Reply:
turns out this is fine 
```rust
async fn fuel_indexer_test_trigger_revert(
        state: web::Data<Arc<AppState>>,
    ) -> impl Responder {
        let _ = state
            .contract
            .methods()
            .trigger_revert()
            .tx_params(tx_params())
            .call()
            .await;

        HttpResponse::Ok()
    }
    ```
    I initially added the match as I was getting error response codes, but it must have been for some other reason.